WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -1250,7 +1253,7 @@ class MySiteViewModel @Inject constructor(
     }
 
     private fun onBloggingPromptShareClick(message: String) {
-        onShare.postValue(Event(message))
+        _onShareBloggingPrompt.postValue(Event(message))
```

Comment:
Since we're on the main thread, would it make sense to use `.value =` instead of `postValue`?

Reply:
That makes sense, but there are a few other methods doing the same thing, I will go ahead and change the ones related to the Blogging Prompts card, so it's not far from the scope of this PR.